controllers/defaults/resources.go

```diff
@@ -124,6 +124,36 @@ var (
 				corev1.ResourceMemory: resource.MustParse("128Mi"),
 			},
 		},
+		"ocs-provider-server": {
```

Comment:
Hope these numbers of backed by some kind of recommendation or testing

Reply:
Yes, these numbers are derived from testing and checked with Leela on this.